packages/bpk-component-card/src/BpkCardWrapper.module.scss

```diff
@@ -19,8 +19,9 @@
 @import '~bpk-mixins/index.scss';
 
 .bpk-card-wrapper {
-  @include bpk-border-radius-md;
   @include bpk-card;
+  @include bpk-border-radius-md;
+  @include bpk-border-lg(var(--background-color), '');
```

Comment:
Looking at the Figma design now it actually seems like the shadow shouldn't be the colour of the wrapper, but the default shadows we support in Backpack

Reply:
Thanks for helping confirmation on this. I have remove shadow effect when hover and use default shadow instead.